lib/src/main/java/graphql/nadel/engine/blueprint/IntrospectionService.kt

```diff
@@ -70,7 +71,7 @@ open class NadelDefaultIntrospectionRunner(schema: GraphQLSchema) : ServiceExecu
                     // This inserts data fetchers for namespaced fields so we can handle their __typename internally
                     getFieldsWithCoordinates(schema.queryType, schema.mutationType, schema.subscriptionType)
                         .filter { (_, field) ->
-                            isNamespacedField(field)
+                            isNamespacedFieldLike(field)
```

Comment:
So this uses the newly extracted code from `NextgenEngine.isOnlyTopLevelFieldTypename`